.github/autolabeler.yml

```diff
@@ -0,0 +1,31 @@
+'accessibility': []
```

Comment:
If we need to list tags with empty patterns for some reason, you missed `patch`

Reply:
no we don't I just extracted the first page worth of tags, I'll clean up the empty ones and the spaces @igor-dv mentioned